libs/application-generic/src/index.ts

```diff
@@ -5,14 +5,15 @@ export * from './decorators';
 export * from './dtos';
 export * from './encryption/index';
 export * from './factories/index';
+export * from './graceful-shutdown/index';
```

Comment:
Extra path not needed

```suggestion
export * from './graceful-shutdown';
```

Reply:
Okay 👍🏼

Reply:
Done